cmd/lightclient/sentinel/sentinel.go

```diff
@@ -174,7 +177,10 @@ func New(ctx context.Context, cfg *SentinelConfig) (*Sentinel, error) {
 	return s, nil
 }
 
-func (s *Sentinel) Start() error {
+func (s *Sentinel) Start(
+	gossipHandler func(*proto.GossipContext) error,
```

Comment:
If possible lets not pass anything to this function, and send all the gossip to only one channel which we can handle outside the Sentinel instead of using a function as a parameter

Reply:
done